profiles.md

```diff
@@ -0,0 +1,234 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This repository allows to manage the profile item linked to a particular eperson. The specific type of item 
+with which the profile is modeled depends on the dspace configuration (by default it is the Person type). If the particular dspace instance doesn't handle the "profile" 
+type then the whole repository is disabled.
```

Comment:
Tiny request...rather than say "then the whole repository is disabled", this should say "then these endpoints are disabled".  The term "repository" doesn't make sense here as it could be misunderstood to mean "all of DSpace".